packages/x-components/tests/e2e/cucumber/mocked-responses.spec.ts

```diff
@@ -0,0 +1,558 @@
+import {
```

Comment:
I'm seeing that we have one Results intercept after the spellcheck:

![image](https://user-images.githubusercontent.com/3502075/142826340-0f567f23-76a3-4a4d-90e6-763080c2d032.png)

Reply:
Also I would try to improve the steps names, as the `...with a known response` is quite confuse, what do you think?

Reply:
The `...with a known response` is there to tell that the response is specified in the test (because we're mocking it). I recall we wrote it this way because we had to avoid using `...with a mocked response`. Do you have any alternative to say the same? I can't think of any 😞 .

Reply:
The thing is that in the `a results API` you're also mocking the response 😆.

I'm the worst with the names so we can think about this one in the future.